src/GraphQL.sln

```diff
@@ -145,6 +145,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.DataLoader", "Graph
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.AotCompilationSample", "GraphQL.AotCompilationSample\GraphQL.AotCompilationSample.csproj", "{789A6AD9-D4F0-44E6-A9BA-57B1BA3A616A}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.StarWars.TypeFirst", "GraphQL.StarWars.TypeFirst\GraphQL.StarWars.TypeFirst.csproj", "{1FBC5401-91F4-4BFA-BBD8-758B857BE03A}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.AotCompilationSample.TypeFirst", "GraphQL.AotCompilationSample.TypeFirst\GraphQL.AotCompilationSample.TypeFirst.csproj", "{16E4BB19-ADBB-472F-BADD-87BC8F4A7C71}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "AotCompilation", "AotCompilation", "{EA03E3DA-9FCC-40CA-BD23-642B297EA3F4}"
```

Comment:
Folder? Maybe just `AOT`?

Reply:
AOT isn’t very descriptive to anyone who doesn’t know what it is already.

Reply:
OK